teaser/project.py

```diff
@@ -245,21 +213,21 @@ def retrofit_all_buildings(
                     raise ValueError("you need to set year_of_retrofit for " "retrofit")
                 iwu_buildings.append(bldg)
 
-        if self.data.used_statistic == "iwu":
+        if self.data == DataClass(construction_data=datahandling.ConstructionData.iwu_heavy):
```

Comment:
what about iwu_light?

Reply:
At this point, my aim was to load the iwu DataClass through the dafault iwu_heavy. So it should be independent of heavy/light, the important point here was that the iwu DataClass appears as a condition in the case differentiation.